tensorboard/webapp/widgets/data_table/data_table_component.scss

```diff
@@ -23,8 +23,7 @@ $_accent: map-get(mat.get-color-config($tb-theme), accent);
   display: table;
   width: 100%;
 
-  .header,
-  .row {
+  .header {
```

Comment:
Is this style still being used?

Reply:
Yes, the header class is used as the "row" for the headers.